inventory_management_system_api/cli/create.py

```diff
@@ -50,8 +50,11 @@ def system_type():
     if system_types_collection.find_one({"value": re.compile(new_system_type_value, re.IGNORECASE)}):
         exit_with_error("A system type with the same value already exists!")
 
+    # Obtain description of the new system type
+    new_system_type_description = Prompt.ask("Please enter the description of the new system type")
+
     # Insert the new system type
-    system_types_collection.insert_one({"value": new_system_type_value})
+    system_types_collection.insert_one({"value": new_system_type_value, "description": new_system_type_description})
```

Comment:
Looking at the database it stores this as an empty string when you dont supply it.

<img width="528" height="104" alt="Image" src="https://github.com/user-attachments/assets/d3c119fa-7a2d-465d-bb68-387e5fb9f521" />

This should really be stored as `null` instead. To do this you would need to pass `None` here.